exercises/space-age/spec/space_age_spec.cr

```diff
@@ -0,0 +1,36 @@
+require "spec"
+require "../src/*"
+
+describe "SpaceAge" do
+  it "age on Earth" do
+    SpaceAge.age("Earth", 1000000000).should eq(31.69)
+  end
+
+  it "age on Mercury" do
```

Comment:
I believe these should be pending :)

Reply:
:( Sorry, I keep forgetting to check them before making a commit. I'll fix this when I get home today.